wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -133,29 +133,76 @@ BeatmapSetID (Integer) is the ID of the beatmap set.
 Difficulty
 ----------
 
-HPDrainRate (Decimal) specifies the HP Drain difficulty.
+The following 4 difficulty properties are expressed in *stars*, where 0 star means very easy, and 10 stars means very hard.
```

Comment:
- `The following 4 difficulty` -> `The following four difficulty`
- `where 0 star means very easy, and 10 stars means very hard.` -> `where 0 stars is very easy while 10 stars is very hard.`